packages/web-app-files/src/mixins/actions/delete.js

```diff
@@ -60,6 +61,16 @@ export default {
             if (this.capabilities?.files?.permanent_deletion === false) {
               return false
             }
+
+            const isProjectSpace = isProjectSpaceResource(this.space)
+            if (
+              isProjectSpace &&
```

Comment:
```suggestion
              isProjectSpaceResource(this.space) &&
```
why assign it to a const? you could use it inline..